### PR TITLE
Filtering logs

### PR DESCRIPTION
**Pull request by tonysnowboardunderthebridge**
Hi @marcopiraccini 

adding the fixing for filtering logs

### CODE REVIEW

File: src/renderer/src/App.jsx [hunk 1]
```diff
@@ -26,13 +26,15 @@ function App ({ path }) {
   const [showModalImportApplication, setShowModalImportApplication] = useState(false)
   const [showModalCreateApplication, setShowModalCreateApplication] = useState(false)
   const featureFlag = isDevMode()
+  const [showErrorComponent, setShowErrorComponent] = useState(false)
 
   const {
     ErrorBoundary,
-    didCatch,
+    // didCatch,
```
Comment (marcopiraccini) [inline src/renderer/src/App.jsx:33]: Why commented? If useless, please remove it

File: src/renderer/src/components/application/application-logs/ApplicationLogs.jsx [hunk 2]
```diff
@@ -71,36 +102,75 @@ const ApplicationLogs = React.forwardRef(({ applicationSelected }, ref) => {
...
+      if (!filtersInitialized) {
+        setFilterLogsByService({ value: newServices[0].value, label: newServices[0].label })
+        setDefaultOptionsSelected({ value: newServices[0].value, label: newServices[0].label })
+        setFilterLogsByLevel(30)
+        setFiltersInitialized(true)
+        return
+      }
+
+      if (filterLogsByLevel || filterLogsByService.value) {
+        console.log('filterLogsByService', filterLogsByService)
```
Comment (marcopiraccini) [inline src/renderer/src/components/application/application-logs/ApplicationLogs.jsx:127]: Pleas remve these console.log

File: src/renderer/src/components/application/application-logs/ApplicationLogs.jsx [hunk 3]
```diff
@@ -71,36 +102,75 @@ const ApplicationLogs = React.forwardRef(({ applicationSelected }, ref) => {
...
+        console.log('filterLogsByLevel', filterLogsByLevel)
+        let founds = [...applicationLogs]
+        founds = founds.filter(log => JSON.parse(log).level >= filterLogsByLevel)
+        if (filterLogsByService.value !== null) {
+          founds = founds.filter(log => {
+            console.log('JSON.parse', JSON.parse(log).name, filterLogsByService)
```
Comment (marcopiraccini) [inline src/renderer/src/components/application/application-logs/ApplicationLogs.jsx:133]: Here too

**Review by marcopiraccini — CHANGES_REQUESTED** [review]
We need to clean up some console.log, then it's GTG.

File: src/renderer/src/components/application/application-logs/ApplicationLogs.jsx (hunk 3 above)
Comment (tonysnowboardunderthebridge) [reply src/renderer/src/components/application/application-logs/ApplicationLogs.jsx]: removed

File: src/renderer/src/components/application/application-logs/ApplicationLogs.jsx (hunk 2 above)
Comment (tonysnowboardunderthebridge) [reply src/renderer/src/components/application/application-logs/ApplicationLogs.jsx]: removed

File: src/renderer/src/App.jsx (hunk 1 above)
Comment (tonysnowboardunderthebridge) [reply src/renderer/src/App.jsx]: removed

**Review by marcopiraccini — APPROVED** [review]
lgtm

**Pull request merged**